tools/run_tests/run_tests.py

```diff
@@ -509,6 +526,7 @@ def __str__(self):
 _WINDOWS_CONFIG = {
     'dbg': 'Debug',
     'opt': 'Release',
+    'gcov': 'Release',
```

Comment:
Shouldn't gcov tests be run with a debug configuration so that they get compiled with `-O0`?